app/controllers/claims_controller.rb

```diff
@@ -4,6 +4,11 @@ class ClaimsController < ApplicationController
 
   after_action :clear_claim_session, if: :submission_complete?
 
+  def new
+    @current_claim = Claim.new(eligibility: StudentLoans::Eligibility.new)
+    render "qts_year"
```

Comment:
Is there a way we can use the page sequence rather than hard-coding this to a specific view?

Reply:
I did consider that, but seeing that it hadn't been done previously (when redirecting etc), I kinda assumed there was a reason for not doing that. Makes sense to give it a go though 👍

Reply:
By delegating to the page sequence, we decouple the controller from specific pages, which means future policies just have to define their own sequence and the controller code doesn't have to change or have conditions.